packages/react-native-webview/src/index.tsx

```diff
@@ -105,4 +105,5 @@ const WebViewComponent = (props: Props, ref: React.Ref<RNWebView<Props>>) => {
 
 export const WebView = forwardRef(WebViewComponent);
 
+// eslint-disable-next-line import/no-default-export
```

Comment:
Removing the default export here would entail forcing users to modify their imports, and I don't this should be the issue to do it.